multivac/fetch.py

```diff
@@ -388,6 +397,7 @@ def download_workflow_run_jobs(workflow_run_id):
             if not job.is_stored:
                 if not args.nologs:
                     job.download_log()
+                    job.check_pat_limit()
```

Comment:
Do you check limits each time you download job data? Why is this necessary? Don't we just spend the limit at double rate this way?
You can fetch it before the loop, store the number and compare it with the lengh of the list, filtered by `job.is_stored`.

Also, what if the limit is out? As I see, the code does not act on this data.